assign_nodes.py

```diff
@@ -237,16 +239,22 @@ def interactive_search_wrapper(g,user_input_file, output_dir, input_type):
 	#Check for existence based on input type
 	exists = check_input_existence(output_dir,input_type)
 	if(exists[0] == 'false'):
-		print('Interactive Node Search')
-		logging.info('Interactive Node Search')
-		#Interactively assign node
-		if input_type == 'annotated_diagram':
-			u = read_user_input(user_input_file[0])
-		if input_type == 'pathway_ocr':
-			u = read_ocr_input(user_input_file)
-		n = unique_nodes(u)
-		s = search_nodes(n,g,u)
-		create_input_file(s,output_dir,input_type)
+            print('Interactive Node Search')
+            logging.info('Interactive Node Search')
+            #Interactively assign node
+            if input_type == 'annotated_diagram':
+                u = read_user_input(user_input_file[0])
+                n = unique_nodes(u)
+            if input_type == 'pathway_ocr':
+                n = []
```

Comment:
After running, I'm getting this error:

Traceback (most recent call last):
  File "/Users/brooksantangelo/Documents/HunterLab/Cartoomics/git/Cartoomics/creating_subgraph_from_KG.py", line 47, in <module>
    main()
  File "/Users/brooksantangelo/Documents/HunterLab/Cartoomics/git/Cartoomics/creating_subgraph_from_KG.py", line 22, in main
    s = interactive_search_wrapper(g, input_file, output_dir, input_type)
  File "/Users/brooksantangelo/Documents/HunterLab/Cartoomics/git/Cartoomics/assign_nodes.py", line 258, in interactive_search_wrapper
    s = search_nodes(n,g,u)
  File "/Users/brooksantangelo/Documents/HunterLab/Cartoomics/git/Cartoomics/assign_nodes.py", line 188, in search_nodes
    examples.loc[examples["target"] == node,"target_label"] = node_label
  File "/Users/brooksantangelo/miniconda3/lib/python3.9/site-packages/pandas/core/frame.py", line 3505, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/Users/brooksantangelo/miniconda3/lib/python3.9/site-packages/pandas/core/indexes/base.py", line 3631, in get_loc
    raise KeyError(key) from err
KeyError: 'target'

What will the target end up being for this diagram?

Reply:
This will create all pairwise paths. However, I just realized that includes redundant pairs (i.e., {source: ACLY, target: CAC}, {source:CAC, target:ACLY}). Will fix tomorrow.